key/src/lib.rs

```diff
@@ -25,6 +25,7 @@ mod network;
 mod password;
 mod platform_address;
 mod random;
+mod seal;
```

Comment:
I thought these names:
* curve25519xsalsa20poly1305
* asymmetric_encryption
* seal

It was hard to choose.

If we follow our convention it should be "curve25519xsalsa20poly1305". We have ed25519 for signature, and x25519 for exchange.
I thought it is too long, but that may be a solution.

We may choose "asymmetric_encryption" since that is what we are doing.

If the name "seal" is ambiguous, how about "asymmetric_encryption"?

Reply:
I think just encryption is ok too.

Reply:
Hmm, I worried that someone may understand it as symmetric encryption.

Let's use `encryption` and change it later when we need symmetric encryption.

Reply:
Ok. Are you going to update the name in this PR? If not, I'll approve this.

Reply:
I'm updating it.

Reply:
Updated it: https://github.com/CodeChain-io/foundry/compare/29ffae7882f098f26ff2a89c827e0998ad724472..38d7936e5a1c131b17e5b2967eb434dc5fb11c64